packages/jest-transform/src/__tests__/script_transformer.test.js

```diff
@@ -46,7 +47,7 @@ jest.mock(
 
     return {
       getCacheKey: jest.fn((content, filename, configStr) => 'ab'),
-      process: (content, filename, config) => `
+      process: (content, filename, config) => jest.requireActual('dedent')`
```

Comment:
Is `dedent` mocked somewhere that you need a `requireActual`?

Reply:
good point. I just used it automatically since I couldn't use `import` at the top 👍